packages/clay-slider/README.mdx

```diff
@@ -6,18 +6,27 @@ packageNpm: '@clayui/slider'
 sibling: 'docs/components/css-slider.html'
 ---
 
-import {Slider} from '$packages/clay-slider/docs/index';
+import {DecadeSlider, Slider} from '$packages/clay-slider/docs/index';
 
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
+-   [Customization](#customization)
 -   [API](#api)
 
 </div>
 </div>
 
+You can use Slider with just 2 props for it's basic usage, `value` and `onValueChange` which is used to set the value of the Slider.
+
 <Slider />
 
+## Customization
```

Comment:
I think we should aim for specific rather than general for our headings. So this could be `Range and Step`